tests/end2end/test_editing_user.py

```diff
@@ -41,7 +41,9 @@ def test_user_can_edit_details_for_existing_user(user, testapp):
     assert '<td>{}</td>'.format(form['username'].value) in res
     assert '<td>{}</td>'.format(form['full_name'].value) in res
     assert '<td>{}</td>'.format(form['active'].checked) in res
-    assert '<td>{}</td>'.format(form['is_admin'].checked) in res
+    assert '<td class="bool-value-{}">{}</td>'.format(
+        format(form['is_admin'].checked).lower(),
+        _('Yes' if form['is_admin'].checked else 'No')) in res
```

Comment:
Apparently, this creates a bogus record in translations file...

```python
#: tests/end2end/test_editing_user.py:46
msgid "Yesis_adminNo"
msgstr ""
```

Will see if I can fix it in upcoming pull request..

Reply:
@oBlissing This fixes it on my computer:

```python
    assert '<td class="bool-value-{}">{}</td>'.format(
        format(form['is_admin'].checked).lower(),
        _('Yes') if form['is_admin'].checked else _('No')
    ) in res
```